backend/src/test/java/ai/verta/modeldb/ExperimentRunTest.java

```diff
@@ -9934,7 +9938,7 @@ public void findExperimentRunsHyperparameterWithStringNumberFloat()
             .addHyperparameters(hyperparameter1)
             .addHyperparameters(
                 KeyValue.newBuilder()
-                    .setKey("C")
+                    .setKey("D")
```

Comment:
we have a known limitation that HP/ Metric with same key need to have values of the same type, hence changed the key for the string HP. This was breaking on Postgres